### PR TITLE
Feature/quality flags

### PR DESCRIPTION
**Pull request by StaudenmannM**
Add quality flags for description and description with gaps. no flag for label and images because they are mandatory data for all cards.

### CODE REVIEW

**Comment by bobman38** [comment]
This one will fix #42

**Comment by bobman38** [comment]
The same as Travis I have an issue with the command `bin/console doctrine:migrations:migrate` on the `postUp` method: maybe we can get rid of Migration files at all ?

Hum and `doctrine:schema:update` doesn't work also 

How have you fix the database on your local environment ? drop all ?

**Comment by StaudenmannM** [comment]
Oops, I forgot to remove my tests and add data in the fixture... Running `bin/console doctrine:migrations:migrate` should be ok to add missing columns and data.

File: src/Controller/CardController.php [hunk 1]
```diff
@@ -66,12 +66,38 @@ public function edit(Nomenclature $nomenclature, Request $request) {
...
+                }
+
+                // test the descripition with gaps tag
+                if($card->getDescriptionWithGaps() != '') {
+                  if($hasDescriptionWithGaps == 'No')
+                    $hasDescriptionWithGaps = 'Yes';
+                  } else {
+                    if($hasDescriptionWithGaps == 'Yes')
+                  $hasDescriptionWithGaps = 'Partial';
+                }
```
Comment (bobman38) [inline src/Controller/CardController.php]: I have an issue with this algorithm...

If first card is empty ('') and all other are provided then it end with a 'Yes' and we wait for a 'Partial'

I think we must count the number of card, count the number of card with empty description and do the math 😄

**Comment by bobman38** [comment]
Following my review I have added one commit can you check it ?

If it's ok you can merge ! Thanks !

**Pull request merged**